code/modules/reagents/newchem/other.dm

```diff
@@ -305,6 +305,47 @@ datum/reagent/fartonium/on_mob_life(var/mob/living/M as mob)
 	return
 
 
+/datum/chemical_reaction/soapification
+	name = "Soapification"
+	id = "soapification"
+	result = null
+	required_reagents = list("liquidgibs" = 10, "lye"  = 10) // requires two scooped gib tiles
+	min_temp = 374
+	result_amount = 1
+
+
+/datum/chemical_reaction/soapification/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	new /obj/item/weapon/soap/homemade(location)
+	return
+
+/datum/chemical_reaction/meatification
+	name = "Meatification"
+	id = "meatification"
+	result = null
+	required_reagents = list("liquidgibs" = 10, "nutriment" = 10, "carbon" = 10)
+	result_amount = 1
+
+/datum/chemical_reaction/meatification/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/meatproduct(location)
+	return
+
+/datum/chemical_reaction/lye
+	name = "lye"
+	id = "lye"
+	result = "lye"
+	required_reagents = list("sodium" = 1, "hydrogen" = 1, "oxygen" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/lye2
+	name = "lye"
+	id = "lye"
+	result = "lye"
+	required_reagents = list("ash" = 1, "water" = 1)
```

Comment:
I wouldn't worry about this one, personally; it's so much more difficult to make that players will almost never go for this outside of burning paper (which would still be more time consuming than the first reaction), and it makes charcoal significantly more annoying and time consuming to make.